SetupContextMenu.ps1

```diff
@@ -111,27 +124,28 @@ $profiles | ForEach-Object {
         
         [void](New-ItemProperty -Path $subItemCMDPath -Name "(default)" -PropertyType String -Value "$env:LOCALAPPDATA\Microsoft\WindowsApps\wt.exe -p `"$profileName`" -d `"%V`"")
 
-        if ($configEntry.icon) {
-            $icoPath = $configEntry.icon
+
+        if ($_.icon) {
+            $icoPath = $_.icon
         }
         elseif(($commandLine -match "^cmd\.exe\s?.*")) {
-            $icoPath = $cmdIcoFileName
+            $icoPath = "$resourcePath$cmdIcoFileName"
```

Comment:
I wonder why you want to set `$resourcePath` here.
This is the second PR point out this change 🤣